pkg/proxy/ipvs/proxier.go

```diff
@@ -1101,6 +1101,15 @@ func (proxier *Proxier) syncProxyRules() {
 	// activeBindAddrs represents ip address successfully bind to DefaultDummyDevice in this round of sync
 	activeBindAddrs := map[string]bool{}
 
+	appliedSvcs, err := proxier.ipvs.GetVirtualServers()
+	if err == nil {
+		for _, appliedSvc := range appliedSvcs {
+			currentIPVSServices[appliedSvc.String()] = appliedSvc
+		}
+	} else {
+		klog.Errorf("Failed to get ipvs services, err: %v", err)
```

Comment:
Should this warrant an early return now since an error from GetVirtualServers impacts every Service sync now?

Reply:
Right. My original thought was letting it proceed trying to insert every IPVS service and destination again but that would be too wasteful, better let it to the next sync.